docs/python/debugging.md

```diff
@@ -125,25 +328,13 @@ You can also rely on a relative path from the workspace root. For example, if th
 "program": "${workspaceFolder}/pokemongo_bot/event_handlers/__init__.py",
 ```
 
-### `pythonPath`
+### `python`
 
 Points to the Python interpreter to be used for debugging, which can be a folder containing a Python interpreter. The value can use variables like `${workspaceFolder}` and `${workspaceFolder}/.venv`.
 
-If not specified, this setting defaults to the interpreter identified in the `python.pythonPath` setting, which is equivalent to using the value `${config:python.pythonPath}`. To use a different interpreter, specify its path instead in the `pythonPath` property of a debug configuration.
+If not specified, this setting defaults to the interpreter identified in the `python.pythonPath` setting, which is equivalent to using the value `${config:python.pythonPath}`. To use a different interpreter, specify its path instead in the `python` property of a debug configuration.
 
-You can specify platform-specific paths by placing `pythonPath` within a parent object named `osx`, `windows`, or `linux`. For example, the configuration for PySpark uses the following values:
-
-```json
-"osx": {
-    "pythonPath": "^\"\\${env:SPARK_HOME}/bin/spark-submit\""
-},
-"windows": {
-    "pythonPath": "^\"\\${env:SPARK_HOME}/bin/spark-submit.cmd\""
-},
-"linux": {
-    "pythonPath": "^\"\\${env:SPARK_HOME}/bin/spark-submit\""
-},
-```
+You can specify platform-specific paths by placing `python` within a parent object named `osx`, `windows`, or `linux`.
```

Comment:
This is true for all properties in the debug configuration - I wonder if we need to mention it for this one specifically. But if we do, it should probably link to https://code.visualstudio.com/Docs/editor/debugging#_platformspecific-properties

Reply:
Given it's not Python specific I removed it.